doc/doxygen/src/ForManagers.dox

```diff
@@ -106,8 +106,10 @@
  * - Implement the @fqref{managerApi.ManagerInterface}
  *   "ManagerInterface" class methods
  *   @fqref{managerApi.ManagerInterface.identifier} "identifier",
- *   @fqref{managerApi.ManagerInterface.displayName} "displayName" and
- *   @fqref{managerApi.ManagerInterface.hasCapability} "hasCapability".
+ *   @fqref{managerApi.ManagerInterface.displayName} "displayName",
```

Comment:
[Fix] We should add the same additions to this doc as the hosts one, eg in "Required for publishing" - "update the implementation of `managementPolicy` _and `hasCapability`_...",

Reply:
Yeah. https://github.com/OpenAssetIO/OpenAssetIO/pull/1141/commits/515e1d9ebebcd4826bcde2307b1c40c968b1accd

Changed a note below about what methods to implement. We've changed the categorisation since this has been written, and I think with the required methods we can just say to implement the publishing capability, which is ideal imo.